crates/ide-completion/src/tests/use_tree.rs

```diff
@@ -273,11 +272,21 @@ pub use $0;
             md bar
             kw crate::
             kw self::
-            kw super::
         "#]],
     );
 }
 
+#[test]
+fn pub_use_tree_super_in_crate_root() {
+    //https://github.com/rust-lang/rust-analyzer/issues/12439
+    check(
+        r#"
+pub use super::$0;
+"#,
```

Comment:
It makes more sense to test that we don't suggest a `super` that jumps out of a crate. The test case you've written here is already tested plenty now as can be seen by the diff in the test fixtures.

So the following test would test my other review part:
```suggestion
        r#"
mod foo {
    pub use super::$0;
}
"#,
```